app/models/interview.rb

```diff
@@ -8,4 +8,24 @@ class Interview < ActiveRecord::Base
   validates :lastname, :firstname, presence: true
   validates :template_id, presence: true
   validates :target_level, presence: true, inclusion: { in: TARGET_LEVELS, message: "%{value} is not a valid level" }
+
+  def self.interview_question_mark_nil_first(interview)
+    mark_nil_first = 1
```

Comment:
Please explain what are instance methods and what are class methods.
what kind of method did you use here?